frontend/public/components/modals/delete-modal.jsx

```diff
@@ -45,17 +45,22 @@ class DeleteModal extends PromiseComponent {
     const {kind, resource} = this.props;
     return <form onSubmit={this._submit} name="form">
       <ModalTitle>Delete {kind.label}</ModalTitle>
-      <ModalBody>
-        Are you sure you want to delete <strong>{resource.metadata.name}</strong>
-        {_.has(resource.metadata, 'namespace') && <span> in namespace <strong>{resource.metadata.namespace}</strong>?</span>}
-        {_.has(kind, 'propagationPolicy') && <div className="co-delete-modal-checkbox">
-          <label className="co-delete-modal-checkbox-label">
-            <input type="checkbox" onChange={() => this.setState({isChecked: !this.state.isChecked})} checked={!!this.state.isChecked} />
-            &nbsp;&nbsp; <span>Delete dependent objects of this resource</span>
-          </label>
-        </div>}
+      <ModalBody className="modal-body co-delete-modal">
+        <span aria-hidden="true" className="co-delete-modal__icon pficon pficon-warning-triangle-o"></span>
+        <div>
+          <p className="lead">Delete {resource.metadata.name}?</p>
```

Comment:
you could use the message dialog component from [pf-react](https://rawgit.com/patternfly/patternfly-react/gh-pages/index.html?knob-With%20Secondary%20Button=true&knob-Show%20Modal=true&knob-Close%20Popover=true&knob-With%20Secondary%20Content=true&knob-animate=true&knob-Show%20Help=true&knob-Icon=info&knob-Field%20Level%20Help%20Content=Please%20specify%20Country%20code%20%3Cbr%3E%20%3Ca%20target%3D%27_blank%27%20href%3D%27https%3A%2F%2Fcountrycode.org%2F%27%3EClick%20here%20for%20a%20list%20of%20Country%20codes%3C%2Fa%3E&knob-With%20Title=true&knob-With%20Primary%20Content=true&knob-On%20Color=primary&knob-Off%20Color=default&knob-With%20Icon=true&selectedKind=patternfly-react%2FCommunication%2FMessage%20Dialog&selectedStory=Error%20Dialog%20Example&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel)

Reply:
That was something discussed and will be an eventual goal. For now, applying the patternfly css  was the decision, mostly because of time.